lib/workers/repository/config-migration/branch/migrated-data.ts

```diff
@@ -1,14 +1,62 @@
 import detectIndent from 'detect-indent';
 import JSON5 from 'json5';
+import prettier from 'prettier';
 import { migrateConfig } from '../../../../config/migration';
 import { logger } from '../../../../logger';
 import { readLocalFile } from '../../../../util/fs';
+import { getFileList } from '../../../../util/git';
 import { detectRepoFileConfig } from '../../init/merge';
 
 export interface MigratedData {
   content: string;
   filename: string;
 }
+interface Indent {
+  amount: number;
+  indent: string;
+  type?: string;
+}
+export async function applyPrettierFormatting(
+  content: string,
+  parser: string,
+  indent: Indent
+): Promise<string> {
+  const prettierConfigFilenames = [
+    '.prettierrc',
+    '.prettierrc.json',
+    '.prettierrc.yml',
+    '.prettierrc.yaml',
+    '.prettierrc.json5',
+    '.prettierrc.js',
+    '.prettierrc.cjs',
+    'prettier.config.js',
+    'prettier.config.cjs',
+    '.prettierrc.toml',
+  ];
+  let prettierExists = (await getFileList()).some((file) =>
+    prettierConfigFilenames.includes(file)
+  );
+  if (!prettierExists) {
+    try {
+      const packageJsonContent = await readLocalFile('package.json', 'utf8');
+      prettierExists ||=
+        packageJsonContent && JSON.parse(packageJsonContent).prettier;
+    } catch {
+      logger.warn('Invalid JSON found in package.json');
+    }
+  }
+
+  if (!prettierExists) {
+    return content;
+  }
+  const options = {
+    parser: parser,
+    tabWidth: indent.amount,
```

Comment:
`amount` can be zero if it can't detect, so we should fallback to 2 then
```suggestion
    tabWidth: indent.amount === 0 ? 2 : ident.amount,
```

Reply:
Yes. Wanted to confirm before adding☺